docs/source/packages.rst

```diff
@@ -52,11 +53,23 @@ The **signac-flow** package allows users to:
     :target: https://pypi.org/project/signac-dashboard/
     :alt: PyPI signac-dashboard
 
-The **signac-dashboard** package allows users to browse **signac**-managed data spaces:
+The **signac-dashboard** package allows users to:
 
+- Browse **signac**-managed data spaces
 - Visualize and analyze job data such as text, images, or video.
 - Share and collaborate on workspace data through a browser-based GUI.
 
+.. rubric:: synced-collections_
+
+The **synced_collections** package defines data structures that allow users to:
+
+- Transparently synchronize Python objects like lists and dicts with an underlying data store
+- Seamlessly translate data between different storage backends with arbitrary data validation
```

Comment:
This sounds like it’s meant to enable copying from JSON to MongoDB or Redis to Zarr. We don’t really have workflows designed around that. We always focus on memory↔️backend.

Reply:
Agreed. Updated.